lib/model/autocomplete.dart

```diff
@@ -145,6 +145,18 @@ class AutocompleteViewManager {
     autocompleteDataCache.invalidateUser(event.userId);
   }
 
+  void handleMessageEvent(MessageEvent event) {
```

Comment:
I am not sure if it is necessary to pass the `event` parameter as it is not used for now. I just followed the code for `handleRealmUserAddEvent(RealmUserAddEvent event)` method.

Reply:
Yeah, following that pattern is good. This is also the pattern you'll find at the call site in `PerAccountStore.handleEvent`.